.rspec

```diff
@@ -0,0 +1,4 @@
+# .rspec
+--require spec_helper
```

Comment:
You can have a global rspec configuration that lives in your home directory. It's usually better to put your global preferences there so you don't have to worry about other people not liking your choices.

Reply:
To piggyback off of Sara's comment, here are how options are overridden based on the precedence of the file location. https://relishapp.com/rspec/rspec-core/v/3-8/docs/configuration/read-command-line-configuration-options-from-files

This implies that one can have multiple `.rspec` files in different locations for different settings.